src/Containers/Doctors/Doctors.js

```diff
@@ -1,7 +1,6 @@
 import React, { useState, useEffect } from 'react';
+import Card from '../../Components/Cards/Cards'
```

Comment:
We might want to change the name of the folders to Card, since the component is named Card

Reply:
Agreed, almost did it.  Waited.  Can fix on my next sweep through.